app/controllers/admin/imported_commitments_controller.rb

```diff
@@ -0,0 +1,53 @@
+class Admin::ImportedCommitmentsController < ActionController::Base
+  layout 'application'
+
+  before_action :user_is_admin
```

Comment:
Should this method/before action be verb-y like `authenticate_user` in the other controllers?

Reply:
changed to authenticate_admin